packages/deno/src/integrations/globalhandlers.ts

```diff
@@ -79,10 +79,16 @@ function installGlobalErrorHandler(client: Client): void {
     data.preventDefault();
     isExiting = true;
 
-    void flush().then(() => {
-      // rethrow to replicate Deno default behavior
-      throw error;
-    });
+    void flush().then(
```

Comment:
Do we need `void` in here?

```suggestion
    flush().then(
```

Reply:
Good catch